src/cpp/shared/scripting/lua_utils.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
```

Comment:
Include guards instead of this? I believe the rest of the project uses guards

Reply:
Yeah; either I'll move the entire project to use this, or I should just use include guards here.